Classes/Systems/ShortcutHandler.swift

```diff
@@ -57,18 +57,23 @@ struct ShortcutHandler {
                                                      localizedSubtitle: nil,
                                                      icon: bookmarkIcon)
         items.append(bookmarkItem)
-        
+
         // Switchuser
-        if sessionManager.userSessions.count >= 2 {
-            let userSession = sessionManager.userSessions[1]
-            if let username = userSession.username {
-                let userIcon = UIApplicationShortcutIcon(templateImageName: "organization")
-                let userItem = UIApplicationShortcutItem(type: Items.switchAccount.rawValue,
-                                                         localizedTitle: NSLocalizedString("Switch Account", comment: ""),
-                                                         localizedSubtitle: username,
-                                                         icon: userIcon,
-                                                         userInfo: ["sessionIndex": 1])
-                items.append(userItem)
+        if sessionManager.userSessions.count > 1 {
+            let focusedUserSession = sessionManager.focusedUserSession
+
+            if let focusedUsername = focusedUserSession?.username {
+                for userSession in sessionManager.userSessions where userSession.username != focusedUsername {
+                    let userSessionIndex = sessionManager.userSessions.index(of: userSession)
+                    let userIcon = UIApplicationShortcutIcon(templateImageName: "organization")
+                    let userItem = UIApplicationShortcutItem(
+                        type: Items.switchAccount.rawValue,
+                        localizedTitle: NSLocalizedString("Switch to \(userSession.username ?? "")", comment: ""),
+                        localizedSubtitle: "From \(focusedUsername)",
+                        icon: userIcon,
+                        userInfo: ["sessionIndex": userSessionIndex ?? 1])
+                    items.append(userItem)
```

Comment:
Does this add an item for _every_ account you're logged into? Is that something we want to do? Not sure I've seen that discussed anywhere before. I kind of like having the one item to switch to the most-recently-used account.

Reply:
Yeah, we should limit this, especially since there is a cap of four items anyway, it's just going to be ignored.

Reply:
So yes, it adds an item for every account that you're logged in. I don't know if you guys have discussed it before or not, but in my opinion is an incomplete feature to switch only to one account when you may have more than 2. Also for me was a bit unclear from which account I was switching on that's why I am showing the `focusedUsername` and where I can switch from there.

In my opinion, it's useful to have this kind of possibility 😊

What do you think? @rnystrom @BasThomas

Reply:
I think we should leave this how it was personally. If you want to switch to other accounts you can do so via settings. I can imagine another UI in app once we have a profile tab too.